cmd/bucket-targets.go

```diff
@@ -91,13 +92,13 @@ func (sys *BucketTargetSys) SetTarget(ctx context.Context, bucket string, tgt *m
 		}
 		vcfg, err := clnt.GetBucketVersioning(ctx, tgt.TargetBucket)
 		if err != nil {
-			if isErrBucketNotFound(err) {
+			if strings.Contains(err.Error(), "The specified bucket does not exist") {
```

Comment:
use `minio.ToErrorResponse(err).Code == "NoSuchBucket"`

Reply:
the error returned isn't of ErrorResponse type, so cannot use your suggestion